api/src/identity-access-management/domain/usecases/import-user-last-logged-at.usecase.js

```diff
@@ -0,0 +1,37 @@
+import { UserLogin } from '../models/UserLogin.js';
+
+/**
+ * @typedef {function} importUserLastLoggedAt
+ * @param {Object} params
+ * @param {boolean} params.dryRun
+ * @param {string} params.userId
+ * @param {Date} params.lastActivity
+ * @param {UserRepository} params.userRepository
+ * @param {UserLoginRepository} params.userLoginRepository
+ * @return {Promise<boolean>}
+ */
+export const importUserLastLoggedAt = async function ({
+  dryRun,
+  userId,
+  lastActivity,
+  userRepository,
+  userLoginRepository,
+}) {
+  const user = await userRepository.get(userId);
+  if (!user || user.hasBeenAnonymised) {
+    return false;
+  }
+
+  const userLogin = await userLoginRepository.findByUserId(userId);
+
+  if (!userLogin) {
+    const newUserLogin = new UserLogin({ userId, lastLoggedAt: lastActivity });
+    if (!dryRun) await userLoginRepository.create(newUserLogin);
+    return true;
+  }
+  if (!userLogin.lastLoggedAt) {
+    userLogin.lastLoggedAt = lastActivity;
+    if (!dryRun) await userLoginRepository.update(userLogin, { preventUpdatedAt: true });
+    return true;
+  }
```

Comment:
Dans la ticket et la description de la PR il est noté : 
> Vérification sur le champ “users-logins.lastLoggedAt” dans users logins
>  * **Si champ “last logged at” est renseigné alors on s’arrête**

Il faudrait soit modifier le ticket et la description de la PR, soit modifier le comportement du code car actuellement le code continue si le champ `lastLoggedAt` est renseigné.